lib/views/after_auth_screens/chat/select_contact.dart

```diff
@@ -38,15 +35,16 @@ class _SelectContactState extends State<SelectContact> {
         ),
       ),
       body: BaseView<SelectContactViewModel>(
-        onModelReady: (model) {
+        onModelReady: (model) async {
           model.initialise();
-          model.getCurrentOrgUsersList();
+          await model.getCurrentOrgUsersList();
```

Comment:
Why was this change necessary ? Since this was the last line of the function and no further line to wait for it, what practical change does having await here make ?

Reply:
I was trying to debug but during testing the organizations were not being shown in the model so when I used await the test was perfectly executed.